rs/offchain/examples/llama/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "llama_bot"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+async-trait = { workspace = true }
+axum = { workspace = true, features = [] }
+candid = { workspace = true }
+clap = { workspace = true, features = ["derive"] }
+dotenv = { workspace = true }
+ic-agent = { workspace = true }
+oc_bots_sdk = { path = "../../../sdk" }
+oc_bots_sdk_offchain = { path = "../../sdk" }
+rand = { workspace = true }
+serde_json = { workspace = true }
+tokio = { workspace = true, features = ["rt-multi-thread"] }
+tower-http = { workspace = true, features = ["cors", "trace"] }
+tracing = { workspace = true }
+tracing-subscriber = { workspace = true }
+serde = { version = "1.0.217", features = ["derive"] }
```

You could do the same as in above I believe,
```suggestion
serde = { workspace = true, features = ["derive"] }
```